### DISCUSSION TITLE
Sprint 2 General Feedback

### DISCUSSION THREAD

**Discussion by 19Si** [The-Basement-Dwellers/DIT-A91906#70] [General]
**This is a School Assessment**, we would appreciate the feedback but please don't give improved code or any code as it will invalidate the assessment.

This is for the **Second Release**.

the general feedback we would prefer:

- what features are not functional, or janky, or unnecessary
- what would be cool to add
- what could we change?

Thanks for checking out our project! :shipit:
[Second Release Link can be viewed from here](https://github.com/The-Basement-Dwellers/DIT-A91906/releases/tag/sprint-2)

**Comment by Monza-Royale** [comment]
you should fix the layers of the bed

I wanna get in the bed not under it

add a tutorial

you can sneak around the ai and kill them before they agro

**Comment by mr-harding** [comment]
Positive:
multiple levels and glowing effect of interactable elements make it easy to know what to do / where to go
movement mechanics work well and allow for different strategies for attacking enemies
shadow and light create unique aesthetic and interesting theme for the game

Constructive:
some issues with layer ordering (power bar on right walls)
As discussed, it would be great to flesh out the game in sprint 3 with more storyline / context. Intro screen and a tutorial level would be great additions.

Overall, excellent progress so far and the game is looking great!

**Comment by taiyotaguchi** [comment]
**what would be cool to add**
- Death animations
- Death screen instead of instantly restarting
- Sound
- Enemy attacking animations

**what could we change?**
- Attack takes too long
- Health bar is too subtle
